packages/scene-composer/src/components/three-fiber/DataOverlayComponent/styles.scss

```diff
@@ -13,14 +13,43 @@
 
 // Container
 .container {
-  overflow: auto
+  overflow: auto;
+  background-color: awsui.$color-background-container-header;
+  border: 1px solid awsui.$color-border-button-normal-disabled;
+  border-radius: 2px;
+  box-shadow: 0px 1px 4px -2px rgba(0, 28, 36, 0.5);
 }
 .panel-container {
-  width: 300px;
-  background-color: awsui.$color-background-container-content
+  width: 210px;
```

Comment:
Non-blocker: We should try to keep use of pixels limited to borders & drop-shadows (amongst a few other edge cases) & implement responsive units such as `rem` or `em`. FEEs should determine standard for this. Once that's been established for our team we'll start blocking for use of `px`.

Reply:
ok. this value is provided by Brenton and is from polaris popover component, so you will want to involve UX for that conversation as well